<launch>
  
  <node name="pvsPhotometricVisualServoing" pkg="ros_dvs_bridge" type="ros_dvs_bridge_pvsVisualServoing"/>
  <param name="cameraTopic" type="string" value="/camera/image_mono"/>
  <param name="robotTopic" type="string" value="/cmd/vel"/> 

  <param name="logs" type="string" value="$(env HOME)/.ros/ros_dvs_bridge/"/>

	<param name="lambda" type="double" value="0.5"/> 
	<param name="focal" type="double" value="0.008"/> 
# In case the sensor is used at full resolution
#	<param name="pixelSize" type="double" value="0.0000052"/> 
# In case the sensor is used at half resolution (e.g. binning x2)
	<param name="pixelSize" type="double" value="0.0000106"/>
	<param name="u0" type="double" value="320"/> 
	<param name="v0" type="double" value="256"/> 
	<param name="sceneDepth" type="double" value="0.5"/> 

	<param name="controlInBaseFrame" type="bool" value="false"/>
#	<param name="cameraPoseTopic" type="string" value="/tf"/>

</launch>

